### PR TITLE
Run gofmt via golangci-lint

### PR DESCRIPTION
**Pull request by ferhatelmas**
Drop custom script since golangci-lint can already do it.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -24,6 +26,7 @@ run:
 
 linters:
   enable:
+    - gofmt
```
Comment (corneliusweig) [inline .golangci.yml]: I think we don't even need this. Looking at `golangci-lint linters | grep goimports`:
```
goimports: Goimports does everything that gofmt does. Additionally it checks unused imports [fast: true, auto-fix: true]
```

**Comment by ahmetb** [comment]
> I find the latter a little easier to understand, but it's not a big deal.

Agreed.

File: .golangci.yml (hunk 1 above)
Comment (ferhatelmas) [reply .golangci.yml]: not exactly correct.

goimports version of golangci doesn't do simplification.

Comment (ferhatelmas) [reply .golangci.yml]: here is the configuration: https://github.com/golangci/golangci-lint/blob/master/pkg/config/config.go#L139

maybe we should suggest adding it then goimports can be used solely.

Comment (corneliusweig) [reply .golangci.yml]: Mm.. looks like goimports does no simplifications: https://github.com/golang/go/issues/21476

**Comment by corneliusweig** [comment]
So I guess this is indeed the best simplification for `gofmt`. From my side
/lgtm

File: .golangci.yml (hunk 1 above)
Comment (ferhatelmas) [reply .golangci.yml]: There was a long standing demand, I thought it was already added but it seems it's still missing.

**Comment by ferhatelmas** [comment]
@ahmetb I believe this is waiting for your review/approval.

**Pull request merged**